src/TorchSharp/DisposeScope.cs

```diff
@@ -154,9 +154,13 @@ public void MoveToOther(DisposeScope? scope, IEnumerable<IDisposable> disposable
         {
             if (this._disposeScopeManager is null)
                 throw new ObjectDisposedException(this.GetType().FullName);
-            foreach (var disposable in disposables) {
-                if (Disposables.Remove(disposable)) {
-                    AddToOther(scope, disposable);
+            if (scope == null) {
+                Detach(disposables);
+            } else {
```

Comment:
Style: in this code base, I believe we usually put the 'else' on the next line.

Reply:
I searched the code base for "} else {" and found 100+ matches. Searching for "else {" found 2 additional matches. The style for this code base is } else {, i.e. keeping the else on the same line as the preceeding closing brace. The two non-conformants  are Tensor.cs:132 and Examples/ResNet.cs:140.

Would you like me to update those two non-conforming files to match style? Then there would be no confusion for future devs. I know it is specifically requested to not supply stylistic only commits, but I can see the value here if you wish to waive that requirement this time.

Reply:
Also the editorconfig says no newline:
https://github.com/dotnet/TorchSharp/blob/b30a60d361470918c4db6dbbd054606ae269182a/.editorconfig#L72

Reply:
I stand corrected.